### PR TITLE
Add info on how to identify apt package source

### PR DESCRIPTION
**Pull request by BanzaiMan**
This resolves https://github.com/travis-ci/apt-package-whitelist/issues/2244

Some polish, such as screenshots, is desirable.

### CODE REVIEW

**Comment by MariadeAnton** [comment]
I'm :+1: to merge this, I think it helps a lot :100:

File: user/installing-dependencies.md [hunk 1]
```diff
@@ -139,6 +139,34 @@ addons:
    - g++-4.8
 ```
 
+#### Identifying the source for missing package
+
+You see a message like this when you add `packages` but it is not found:
```
Comment (plaindocs) [inline user/installing-dependencies.md]: Can we clarify in the build log.

**Comment by plaindocs** [comment]
@BanzaiMan if you want to explicitly assign to me when you're done with a PR I'll make sure they get in faster than this. Or should I just assume that PR's you make are ready from your perspective?

**Comment by MariadeAnton** [comment]
@BanzaiMan could you update the PR clarifying what @plaindocs mentioned or sync the PR (again) with gh-pages? Thank you!!

File: user/installing-dependencies.md (hunk 1 above)
Comment (BanzaiMan) [inline user/installing-dependencies.md]: I am not sure what you mean by this. With an example build log, or what? There are many lines between `Installing APT Packages (BETA)` and `E: Unable to locate package libcxsparse3.1.2`, and it is not practical to include them all.

**Comment by plaindocs** [comment]
@BanzaiMan @MariadeAnton please check and merge.

**Comment by MariadeAnton** [comment]
💃 this is super helpful!

**Pull request merged**